### ISSUE TITLE
 App with YouTube, installed from Google Play shows white screen, but if install without Google Play - everything is working.

### ISSUE THREAD

**Issue by sneronovAdviqo** [PierfrancescoSoffritti/android-youtube-player#500]
I have an Android App. The App provides video stream by YouTube.
The Application is available for Android Phones and Android TV
compileSdkVersion and targetSdkVersion are 28, minSdkVersion is 19.
Library that is used to show YoutTube stream is com.github.PierfrancescoSoffritti:AndroidYouTubePlayer:10.0.4 that is using WebView to show youtube video.
App is signed and uploaded to Google Play store.

The App is downloading, installing from the Google Play and starting with NO problem.
When the app working on smartphones - there is no problems generally and problem with video. But,
Issue: when start activity with YouTube stream on Android TV, the screen is becoming white and nothing is happening. Sound is not existing as well.

What I tried:
- Smart Phones: everything is working well
- Installation through Android Studio: works well
- Installation by APK file (signed) directly on Android TV: works well
- Downloaded current released APK from Google Play and direct installation on Android TV: WORKED WELL!

How it can be possible, that the App from the same APK file works not correct if install it from Google Play?

**Comment by PierfrancescoSoffritti** [comment]
I don't know, it sounds odd. Are you testing always on the same Android TV device?

**Comment by sneronovAdviqo** [comment]
Yes, tested only on the same device as on emulators android TV they do not have play market.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ismailtosun** [comment]
same problem. When i download my app on play store videos not working.
But when i run my app in emilator using android studio its working fine.
